src/app/public/modules/checkbox/checkbox.component.spec.ts

```diff
@@ -1,14 +1,17 @@
 import {
+  AfterViewInit,
   ChangeDetectionStrategy,
   ChangeDetectorRef,
   Component,
-  DebugElement
+  DebugElement,
+  ViewChild
 } from '@angular/core';
 
 import {
   ComponentFixture,
   async,
-  TestBed
+  TestBed,
+  fakeAsync
```

Comment:
Imports should be listed alphabetically.

Reply:
Noted! fixed in last commit